Makefile

```diff
@@ -79,7 +79,7 @@ test-unit: setup-envtest ## Run the unit tests
 
 FOCUS := $(if $(TEST),-focus "$(TEST)")
 test-e2e: ginkgo ## Run the e2e tests
-	$(GINKGO) -v -trace -progress $(FOCUS) test/e2e
+	$(GINKGO) -trace -progress $(FOCUS) test/e2e
```

Comment:
Mentioned this in slack: but it could be nice to have a know like the FOCUS variable that toggles the -v flag.

Reply:
I propose that if we take `-v` out of the default command, FOCUS should add it back in, since presumably if a specific test is specified the verbosity should be turned up. So 
`FOCUS := $(if $(TEST), -v -focus "$(TEST)")`